app/Jobs/SendPasswordResetToCustomerIo.php

```diff
@@ -65,24 +66,16 @@ public function getUrl()
      *
      * @return void
      */
-    public function handle()
+    public function handle(CustomerIo $customerIo)
     {
-        // Rate limit Blink/Customer.io API requests to 10/s.
+        // Rate limit Customer.io API requests to 10/s.
         $throttler = Redis::throttle('customerio')->allow(10)->every(1);
-        $throttler->then(function () {
-            $payload = [
+        $throttler->then(function () use ($customerIo) {
+            $customerIo->trackEvent($this->user, 'track_call_to_action_email', [
```

Comment:
Shouldn't this be `call_to_action_email`?
```suggestion
            $customerIo->trackEvent($this->user, 'call_to_action_email', [
```

Reply:
Oh, thanks for catching that. I got confused by [this line](https://github.com/DoSomething/blink/blob/dfe6d5ef138bd2fb3d66a105d626f4ab6f62d6c5/src/workers/CustomerIoCallToActionEmailWorker.js#L9) in Blink. Is [this](https://github.com/DoSomething/blink/blob/dfe6d5ef138bd2fb3d66a105d626f4ab6f62d6c5/src/messages/CallToActionEmailMessage.js#L12) where the actual event name is set?

(I want to make sure I'm looking in the right place in the future!)

Reply:
Yup, that's where the event name is set. Later on in that file, it's used to send the Customer.io request:
https://github.com/DoSomething/blink/blob/dfe6d5ef138bd2fb3d66a105d626f4ab6f62d6c5/src/messages/CallToActionEmailMessage.js#L37-L41